blog/_posts/2025-07-29-summer-student-course-2025.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "ROOT Summer Student Course 2025 Recap and New Video Recording"
+layout: archive
+author: Marta Czurylo
+---
+
+As every summer, we organized a number of ROOT courses for a few groups of summer students - we held five sessions with around 30 to 40 participants each, including one session together with HSF/IRIS-HEP Python for Analysis Trainings and one as part of Openlab Workshops.
+
+<center>
+    <img
+    src="{{'/assets/images/summer-student-course-2025-trainers.png' | relative_url}}"
+    alt="2025 Summer Student Course at CERN."
+    style="width: 80%" width="1600" height="749" />
+</center>
+
+The sessions were held in person at CERN. Students were learning about the basics of ROOT. We introduced topics like histograms, graphs and functions, including an example of using our new [Unified Histogram Interface](https://root.cern/blog/uhi-for-root/). We discussed the basics of parameter estimation and showed how to interact with the ROOT files. During the last part of the course, we introduce the RDataFrame and discussed many useful functions and features any particle physics analyst should know.
+
+For those of you who couldn’t make it to the courses in person, we prepared a video recording, which is available [here](https://videos.cern.ch/record/2301866). As the whole course is taught in Python using the Jupyter notebooks, we recommend that you follow along while watching the video. The course material is available [in this GitHub repository](https://github.com/root-project/student-course/tree/25.07) and all instructions are given in the README at the bottom of the page. For the best possible experience, we recommend that you use ROOT version 6.36 or higher.
+
+We hope the video can be of use to anyone starting with ROOT. If you have any feedback on what could be improved or added in a more advanced course, please don’t hesitate to comment below or contact us otherwise. If you are teaching ROOT at your institution, please feel free to use this training material in your course. We are also happy to collaborate, get your feedback and discuss any new ideas.
```

Comment:
```suggestion
We hope the video can be of use to anyone starting with ROOT. If you have any feedback on what could be improved or added in a more advanced course, please don’t hesitate to comment below or contact us otherwise. If you are teaching ROOT at your institution, please feel free to use this training material in your course. We would extremely appreciate your feedback on the course and we would be happy to discuss further collaboration, so do not hesitate to reach out to us!
```